doc/apiref.md

```diff
@@ -47,7 +47,7 @@ process to force reloading configuration files. See
     restart_puppet_server(master)
     ```
 
-### `get_manifests_path`
+### `get_manifests_path(host)`
```

Comment:
The options hash is only documented in the examples. It would be nice to have a full description of what options are available.